src/core/dbus-job.c

```diff
@@ -137,7 +137,7 @@ const sd_bus_vtable bus_job_vtable[] = {
         SD_BUS_METHOD("GetBefore", NULL, "a(usssoo)", bus_job_method_get_waiting_jobs, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_PROPERTY("Id", "u", NULL, offsetof(Job, id), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("Unit", "(so)", property_get_unit, 0, SD_BUS_VTABLE_PROPERTY_CONST),
-        SD_BUS_PROPERTY("JobType", "s", property_get_type, offsetof(Job, type), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("JobType", "s", property_get_type, offsetof(Job, original_type), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
i am not sure how i feel about this. since we merge jobs of different types into one following some rules, we will still change the types of already queued jobs even if this change is made, and there's no 1:1 relationship of the job type you enqueue and the type of the job you get returned for it.

Given that that is that way, I wonder if there's really such a major value of making a job stick to the same type across the restart/start conversion if we can't keep it during job merging either?